docs/enhancements/feature-enablement.md

```diff
@@ -0,0 +1,78 @@
+---
+title: feature-enablement
+authors:
+  - "@nmagnezi"
+creation-date: 2022-05-02
+last-updated: 2022-05-02
+---
+
+# Feature Enablement
```

Comment:
Let's make the title more descriptive.  How about "SaaS Per-Organization Feature Access"?

Reply:
Done